README.md

```diff
@@ -86,7 +86,7 @@ See [server demo](example) and [browser demo](https://github.com/bcherny/json-sc
 | bannerComment | string | `"/* tslint:disable */\n/**\n* This file was automatically generated by json-schema-to-typescript.\n* DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,\n* and run json-schema-to-typescript to regenerate this file.\n*/"` | Disclaimer comment prepended to the top of each generated file |
 | cwd | string | `process.cwd()` | Root directory for resolving [`$ref`](https://tools.ietf.org/id/draft-pbryan-zyp-json-ref-03.html)s |
 | declareExternallyReferenced | boolean | `true` | Declare external schemas referenced via `$ref`? |
-| enumGenType | string | `default` | Defines how enums should be generated: default, const, literal, dts |
+| enumGenType | string | `const` | Defines how enums should be generated: default, const, literal, dts |
```

Comment:
This is a bit pedantic, but it's pretty weird for the default not to be `default`. Do you want to give it a more informative name?

Reply:
Taking suggestions! What would be a good term for `non-const`? 🙃

Reply:
Yep, I can't think of anything better. I found at least one legitimate reference to that term:
https://2ality.com/2020/01/typescript-enums.html#compiling-non-const-enums